data/org.freedesktop.portal.Settings.xml

```diff
@@ -73,8 +73,25 @@
       @value: The value @key is set to.
 
       Reads a single value. Returns an error on any unknown namespace or key.
+
+      Deprecated for ReadOne. The value argument incorrectly returns the value inside two variants instead of one.
     -->
     <method name='Read'>
+      <annotation name="org.freedesktop.DBus.Deprecated" value="true"/>
+      <arg name='namespace' type='s'/>
+      <arg name='key' type='s'/>
+      <arg name='value' direction='out' type='v'/>
+    </method>
+
+    <!--
+      ReadOne:
+      @namespace: Namespace to look up @key in.
+      @key: The key to get.
+      @value: The value @key is set to.
+
+      Reads a single value. Returns an error on any unknown namespace or key.
```

Comment:
Perhaps "Reads a single value, similar to the standard `org.freedesktop.DBus.Properties.Get` method" or something, to emphasize that they're in the same encoding?

Reply:
I copied the "may be any valid DBus type" from the org.freedesktop.DBus.Properties documentation which hopefully indicates what this variant is expected to contain.